internal/venice-consumer/src/main/java/com/linkedin/venice/client/consumer/VeniceChangelogConsumer.java

```diff
@@ -0,0 +1,58 @@
+package com.linkedin.venice.client.consumer;
+
+import com.linkedin.venice.annotation.Experimental;
+import com.linkedin.venice.pubsub.api.PubSubMessage;
+import java.util.Collection;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+
+/**
+ * Venice change capture consumer to provide
+ * @param <K>
+ * @param <V>
+ */
+@Experimental
+public interface VeniceChangelogConsumer<K, V> {
```

Comment:
Do we need a `close` method?

What is the meaning of `unsubscribe` or `unsubscribeAll`? Remove all the local state? And how would the customer resume the subscription from the previous checkpoint?

Reply:
I think we should have a `close` method. For `unsubcribe` or `unsubscribeAll`,  are for remove topic partitions. For resume we can either re-subscribe for specific offset checkpoint or adding resume afterwards. At this moment, just want to make it simple.